twenty-first/src/util_types/merkle_tree.rs

```diff
@@ -842,7 +911,8 @@ pub mod merkle_tree_test {
     #[proptest(cases = 30)]
     fn requesting_inclusion_proof_for_nonexistent_leaf_fails_with_expected_error(
         #[strategy(arb())] tree: MerkleTree,
-        #[filter(#leaf_indices.iter().any(|&i| i > #tree.num_leafs()))] leaf_indices: Vec<usize>,
+        #[filter(#leaf_indices.iter().any(|&i| i > (#tree.num_leafs() as MerkleTreeLeafIndex)))]
```

Comment:
`.num_leafs()` already returns type `MerkleTreeLeafIndex`.

```suggestion
        #[filter(#leaf_indices.iter().any(|&i| i > #tree.num_leafs()))]
```